sdks/sentry.java.android

```diff
@@ -0,0 +1 @@
+./packages/maven/sentry-android
```

Comment:
@untitaker do you know why all the other files are `../packages` and this one generated as `./packages`? like 1 dot instead of 2 dots

Reply:
Wrong docs... going to fix that.